### PR TITLE
Feature/provide i18n

### LINKED ISSUE #11: Provide multilanguage support
Internationalization (i18n) should be provided.

At least we should support locales (and formats) for:
- Catalan
- Spanish
- English

### PR DESCRIPTION
**Pull request by XaviTorello**
## Description

It provides the initial translation files for EN, CA and ES

If someone want to contribute, we've deployed Weblate at https://weblate.geekscat.org/projects/anem-per-feina/ getting translations from this repo :)

Fix #11

Related to #88

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,7 +35,7 @@ We forked the open source project [django-job-portal](https://github.com/manjuru
 
 **Anem per feina** is a collaborative effort where everybody is more than welcome to contribute, no experience is required!
 
-Have a look at [CONTRIBUTING.md](CONTRIBUTING.md) file that describes the process to submit a contribution, and come say hi 👋👋 to [GeeksCat slack](https://geekscat.slack.com) where most contributors hang out.
+Have a look at [CONTRIBUTING.md](CONTRIBUTING.md) file that describes the process to submit a contribution, support translations, and come say hi 👋👋 to [GeeksCat slack](https://geekscat.slack.com) where most contributors hang out.
```
Comment (ytturi) [inline README.md:38]: I think this change is not required. Supporting translations itself is a contribution.

If you want to specify that in order to support the translations you should read the contribution guide first, I think you should probably add a new sentence for it. As it is written right now it looks a bit weird.

**Review by ytturi — CHANGES_REQUESTED** [review]
Just a small grammar nip. Everything else lgtm 👍

File: README.md (hunk 1 above)
Comment (XaviTorello) [reply README.md]: Thanks @ytturi I'll, fixed :)

**Review by francescarpi — APPROVED** [review]
Well done

**Pull request merged**